back/src/Controller/IoSocketController.ts

```diff
@@ -159,6 +159,12 @@ export class IoSocketController {
                         }
 
                         const userUuid = await jwtTokenManager.getUserUuidFromToken(token);
+                        const client = socketManager.searchClientByUuid(userUuid);
+                        //if client socket exist, close connexion
+                        if(client){
+                            console.info(`Close connexion, client with uuid : ${userUuid} already exist !`);
+                            socketManager.leaveRoom(client);
+                        }
```

Comment:
We need to change this to the room level (when scaling is merged).
Indeed, if we have many servers, we cannot rely on `socketManager.searchClientByUuid` to find a user.

We cannot reliably know if a user is logged or not across the cluster but we can prevent someone from joining the same room several times.